aikau/src/test/resources/alfresco/documentlibrary/SearchListTest.js

```diff
@@ -27,303 +27,332 @@ define(["intern!object",
+   //          .end()
+   //          .findByCssSelector("#SET_MULTIPLE_SEARCH_DATA_2")
+   //             .click()
+   //          .end()
+   //          .getLastPublish("ALF_SEARCH_REQUEST")
+   //          .then(function(payload) {
+   //             assert.propertyVal(payload, "term","testTerm3", "Search term not set appropriately from hash change");
+   //             assert.propertyVal(payload, "filters", "filter1,filter2,filter3", "Facet filters not set appropriately from hash change");
+   //          });
+   //       },
 
-      "Test search result count publishes correctly": function() {
-         // Click the button to generate a fake search request response...
-         return browser.findByCssSelector("#PUBLISH_SEARCH_RESULTS")
-            .click()
-         .end()
-         // Check that facet data is published... NOTE: TOPIC THROWS AN ERROR IN TESTING - MUST BE THE @
-         .getLastPublish("ALF_FACET_RESULTS_qname1")
-         .then(function(payload) {
-            assert.deepPropertyVal(payload, "facetResults.test", 3, "Search result facet data not published");
-         })
-         // Check that search result count is published...
-         .getLastPublish("ALF_SEARCH_RESULTS_COUNT")
-         .then(function(payload) {
-            assert.propertyVal(payload, "count", 3, "Search resultset size not published");
-         })
-         // Test that Number of search results is as expected.
-         .findAllByCssSelector(".alfresco-search-AlfSearchResult")
-            .then(function(elements) {
-               assert.lengthOf(elements, 3, "Number of results expected is 3, actual results displayed is: " + elements.length);
-            });
-      },
+   //       "Test search result count publishes correctly": function() {
+   //          // Click the button to generate a fake search request response...
+   //          return browser.findByCssSelector("#PUBLISH_SEARCH_RESULTS")
+   //             .click()
+   //          .end()
+   //          // Check that facet data is published... NOTE: TOPIC THROWS AN ERROR IN TESTING - MUST BE THE @
+   //          .getLastPublish("ALF_FACET_RESULTS_qname1")
+   //          .then(function(payload) {
+   //             assert.deepPropertyVal(payload, "facetResults.test", 3, "Search result facet data not published");
+   //          })
+   //          // Check that search result count is published...
+   //          .getLastPublish("ALF_SEARCH_RESULTS_COUNT")
+   //          .then(function(payload) {
+   //             assert.propertyVal(payload, "count", 3, "Search resultset size not published");
+   //          })
+   //          // Test that Number of search results is as expected.
+   //          .findAllByCssSelector(".alfresco-search-AlfSearchResult")
+   //             .then(function(elements) {
+   //                assert.lengthOf(elements, 3, "Number of results expected is 3, actual results displayed is: " + elements.length);
+   //             });
+   //       },
```

Comment:
Do we need to commit such a massive amount of commenting-out to version control? Can't we just delete it?